pkgs/tools/package-management/cargo-edit/default.nix

```diff
@@ -15,15 +14,25 @@ rustPlatform.buildRustPackage rec {
 
   cargoSha256 = "1h1sy54p7zxijydnhzvkxli90d72biv1inni17licb0vb9dihmnf";
 
-  nativeBuildInputs = [ pkg-config ];
-  buildInputs = [ openssl ] ++ lib.optionals stdenv.isDarwin [ libiconv darwin.apple_sdk.frameworks.Security ];
+  nativeBuildInputs = [
+    pkg-config
+    xcbuild # the cc crate attempts to run xcbuild
+  ];
+  buildInputs = [ openssl zlib ] ++ lib.optionals stdenv.isDarwin [
+    libiconv
+    darwin.apple_sdk.frameworks.Security
+  ];
 
   doCheck = false; # integration tests depend on changing cargo config
 
   meta = with lib; {
-    description = "A utility for managing cargo dependencies from the command line";
+    description =
+      "A utility for managing cargo dependencies from the command line";
     homepage = "https://github.com/killercup/cargo-edit";
-    license = with licenses; [ asl20 /* or */ mit ];
+    license = with licenses; [
+      asl20 # or
+      mit
+    ];
```

Comment:
Are these changes necessary? Did nixpkgs-fmt make them?

Reply:
yes, I just ran the formatter.

Reply:
Strange, I don't get that formatting with `nixpkgs-fmt`.